src/Parsers/ASTCreateQuery.h

```diff
@@ -91,6 +91,8 @@ class ASTCreateQuery : public ASTQueryWithTableAndOutput, public ASTQueryWithOnC
         return removeOnCluster<ASTCreateQuery>(clone(), new_database);
     }
 
+    bool isView() { return is_view || is_materialized_view || is_live_view; }
```

Comment:
Now we have to rename `is_view` to `is_ordinary_view` because otherwise the meaning of `is_view` field is not consistent with the behaviour of `isView` function.

Also you can rename `isView` to `isAnyView`.

Reply:
ok. I think it's better to rename `is_view` to `is_ordinary_view` because it's more clearer to distinguish different kinds of views. `view` is more generic, can be `ordinary`, `materialized` , `live` or more in the future.

Reply:
> Now we have to rename `is_view` to `is_ordinary_view` because otherwise the meaning of `is_view` field is not consistent with the behaviour of `isView` function.
> 
> Also you can rename `isView` to `isAnyView`.

ok. I think it's better to rename is_view to is_ordinary_view because it's more clearer to distinguish different kinds of views. view is more generic, can be ordinary, materialized , live or more in the future.